.zuul.yaml

```diff
@@ -102,6 +102,14 @@
     name: semaphore-container-images-kolla-push-caracal-aarch64
     max: 1
 
+- semaphore:
+    name: semaphore-container-images-kolla-push-dalmatian
```

Comment:
I would prefer to stop using the codenames and just go for "2024.2" everywhere, will propose a followup.